### PR TITLE
Controller test

### PR DESCRIPTION
**Pull request by brendan-bondurant**
This introduces testing for the controller. All of the 11 tests are passing. 

InMemory was added to make the controller unit tests work

The code runs fine locally, and I think everything should be good to go

### CODE REVIEW

File: net6_bread_be/appsettings.Development.json [hunk 1]
```diff
@@ -4,6 +4,9 @@
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "BreadDbConnection": "Host=localhost;Port=5433;Username=postgres;Password=password123;Database=BreadDB"
```
Comment (EricBelongea) [inline net6_bread_be/appsettings.Development.json:10]: Is your local db on pgAdmin called "BreadDB" or "net6_bread_be"?

File: net6_bread_be/appsettings.json [hunk 2]
```diff
@@ -5,6 +5,8 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "BreadDbConnection": "Host=localhost;Port=5433;Username=postgres;Password=password123;Database=BreadDB"
```
Comment (EricBelongea) [inline net6_bread_be/appsettings.json:10]: Is your local db on pgAdmin called "BreadDB" or "net6_bread_be"?

File: net6_bread_be/net6_bread_be.csproj
Comment (EricBelongea) [inline net6_bread_be/net6_bread_be.csproj:1]: Thank you for walking me through what packages you got. I added the Microsoft.EntityFrameworkCore.InMemory and that solved all of my testing issues.

File: net6_bread_be_tests/Controllers/bread_controller_tests.cs
Comment (EricBelongea) [inline net6_bread_be_tests/Controllers/bread_controller_tests.cs:1]: Nice controller tests!

File: net6_bread_be/Models/Bread.cs [hunk 3]
```diff
@@ -11,5 +11,8 @@ public class Bread
         public string Recipe { get; set; } = "";
         [Required(ErrorMessage = "Description is required")]
         public string Description { get; set; } = "";
-	}
+        [Required(ErrorMessage = "Country ID is required")]
+        public int CountryId { get; set; }
```
Comment (EricBelongea) [inline net6_bread_be/Models/Bread.cs:15]: I added this as well.

File: net6_bread_be/Controllers/BreadController.cs
Comment (EricBelongea) [inline net6_bread_be/Controllers/BreadController.cs:1]: Nice ControllerActions and error handling.

**Review by EricBelongea — APPROVED** [review]
This looks good to me!!

File: net6_bread_be/appsettings.json (hunk 2 above)
Comment (brendan-bondurant) [reply net6_bread_be/appsettings.json]: I fixed this, it wasn't pushed up but it is now

File: net6_bread_be/appsettings.Development.json (hunk 1 above)
Comment (brendan-bondurant) [reply net6_bread_be/appsettings.Development.json]: I fixed this, it wasn't pushed up but it is now

**Pull request merged**